### ISSUE TITLE
small network detections

### ISSUE THREAD

**Issue by rbgreenway** [NVIDIA/retinanet-examples#243]
I'm running a very small network (128x256 pixels) to attempt OCR on an extracted license plate image.  No luck at all. The network appears to train fine, but when I test with "odtk infer", I basically get no good detections.  Also, the validation after training reports  "No Detections".  When I say that the network appears to train fine, just by observing the loss curves in tensorboard.

I've checked my training data/annotations, and it looks fine.  I've run the anchor optimization from 
[(https://github.com/martinzlocha/anchor-optimization)]  giving me:
```
Final best anchor configuration
State: 0.13516
Ratios: [0.578, 1.0, 1.729]
Scales: [0.616, 0.737, 1.026]
```
I changed the ratios and scales used in model.py, Model.__init__ function to these values.  I'm not sure what the "State" value reported from this optimization is, so not sure if I need to change something.

So my questions are as follows:
1) Will RetinaNet work on an input image this small?
2) If so, what modifications should I make to optimize for detecting characters that are around 35x45 pixels in size?

Thanks for any insight.

FYI...here's my training command:
`odtk train ocr.pth --images /data/synthesized_single_folder/train/00000 --annotations /data/synthesized_single_folder/train.json --backbone ResNet50FPN --lr 0.00005 --fine-tune retinanet_rn50fpn.pth --val-images /data/synthesized_single_folder/test/00000 --val-annotations /data/synthesized_single_folder/test.json --classes 35 --jitter 128 128 --resize 128 --max-size 256 --augment-brightness 0.01 --augment-contrast 0.01 --augment-hue 0.002 --augment-saturation 0.01 --batch 16 --regularization-l2 0.0001 --iters 10000 --val-iters 5000 --logdir /data/log`

and here's my inference command:
`odtk infer ocr.pth --images=/data/test  --annotations=/data/test/test.json  --resize 128 --max-size 256`

**Comment by yashnv** [comment]
Please use branch 20.03 or follow this https://github.com/NVIDIA/retinanet-examples/issues/219#issuecomment-668883353 if using master branch and 20.06

**Comment by yashnv** [comment]
Feel free to ask questions here. Closing this now due to inactivity.
